ui/components/ui/icon/expend-icon.component.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function ExpendIcon({
```

Comment:
- Should be `ExpandIcon`?
- expend vs expand

Reply:
fixed in 3dc2a64936a9936bef78b9bfbf88a1373ed58d8e

Reply:
- Please also change the file name to `expand-icon.component.js`

Reply:
Will open issue and fix in the future